src/index.html

```diff
@@ -0,0 +1,210 @@
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I’m Daniel Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="intro2">LET’S CONNECT</a>
+        <ul class="social">
+            <li class="social-icon twitter">
+                <a><img src="../assets/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a><img src="../assets/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="../assets/m.png" alt="m"></a>
+            </li>
+            <li class="social-icon github">
+                <a><img src="../assets/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="../assets/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
+    </section>
+    <section id="portfolio" class="project-section">
+
+    </section>
+    <section id="about" class="about">
+        <div class="left">
+            <h2 class="about-me"> About Myself</h2>
+            <p class="intro1">
+                Hello I’m a software developer! I can help you build a
+                product , feature or website Look through some of my
+                work and experience! If you like what you see and have
+                a project you need coded, don’t hestiate to contact me.
+            </p>
+            <h2 class="intro2">LET’S CONNECT</h2>
+            <ul class="social">
+                <li class="social-icon">
+                    <img src="../assets/github.png" alt="github">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/Linkedin.png" alt="Linkedln">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/fingers.png" alt="two fingers">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/twitter.png" alt="twitter">
+                </li>
+
+                <li class="social-icon">
+                    <img src="../assets/m.png" alt="m">
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly ensure that the font-size, and font-weight of the about me texts matches the figma design. Also ensure that the items are aligned all together to the left and they take up 60% of the entire width of the about section.

1. Check the CSS properties on the Figma design so you can use the same in your own design so that the designs can match.

2. To make sure that the let part of the about section takes up 60% of the entire width, I encourage you to use flexbox, so you can use flex property I,e` flex: 60%` to align the container to take 60% of the entire section width.

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222930003-e5b26f4e-9728-4834-ae01-1b58608ed3b9.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222930028-776ff187-d073-45fe-a56c-461d0a207e10.png)

Reply:
Done!